assets/css/_properties_list.scss

```diff
@@ -28,5 +28,9 @@
   .m-properties-list__property--last-login & {
     color: $color-font-grey;
     font-size: 0.75rem;
+
+    .m-search-results__card--new & {
```

Comment:
Either `_properties_list.scss` needs a `.m-search-results` selector, or `_search_results.scss` needs a `.m-properties-list` selector. I think I'd rather this be in `_search_results.scss` and have the parent know the details about its child, than to have it here where the child would need to know details about its parent.

Reply:
Yeah, I went back and forth, but I agree with you, that's better.